engine/events/map_name_sign.asm

```diff
@@ -242,7 +242,7 @@ PlaceMapNameFrame:
 	ret
 
 .FillTopBottom:
-	ld c, 5
+	ld c, ((SCREEN_WIDTH - 2) - 2) / 4 + 1
```

Comment:
This should be `(SCREEN_WIDTH - 2) / 4 + 1`.

Reply:
That's what I did at first, but I thought that wouldn't make it clear as to why that `+1` exists. The first `-2` is to specify the boundaries of the function - we're working from x position `1` to x position `SCREEN_WIDTH - 1`, the second `-2` is for the first iteration, which fills the first two tiles. Of course, this won't work for `SCREEN_WIDTH` values that aren't divisible by 4, but so wouldn't what you're proposing (and it really doesn't matter in this case)...
I think I'll revert it, since that extra `-2` might be adding unnecessary confusion.